src/fhir/CarePlan.DCW.CogLevel1.NoDementiaDx.json

```diff
@@ -0,0 +1,75 @@
+{
+    "resourceType": "CarePlan",
+    "id": "TBD",
+    "description": "DCW cognition domain, level 1 (which is only for patients who don't have a dementia Dx)",
+    "status": "active",
+    "intent": "order",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://snomed.info/sct",
+                    "code": "719091000000102",
+                    "display": "Questionnaire"
+                }
+            ],
+            "text": "Questionnaire"
+        }
+    ],
+    "subject": {
+        "reference": "Patient/TBD",
+        "display": "Doe, John TBD"
+    },
+    "activity": [
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/[ID for ecog12]"
+                ],
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://snomed.info/sct",
+                            "code": "719091000000102",
+                            "display": "Questionnaire"
+                        }
+                    ],
+                    "text": "Everyday Cognition (ECOG) 12 (short form) questionnaire"
+                },
+                "status": "active",
+                "scheduledTiming": {
+                    "repeat": {
+                        "frequency": 1,
+                        "period": 1,
+                        "periodUnit": "d"
+                    }
+                }
+            }
+        },
+        {
+            "detail": {
+                "instantiatesCanonical": [
+                    "Questionnaire/[ID for Mini-Cog]"
+                ],
+                "code": {
+                    "coding": [
+                        {
+                            "system": "http://snomed.info/sct",
+                            "code": "719091000000102",
+                            "display": "Questionnaire"
+                        }
+                    ],
+                    "text": "Mini-Cog"
+                },
+                "status": "active",
```

Comment:
@mcjustin When I tried to post this careplan JSON to smart health it, I got this error: `Unknown CarePlanActivityStatus code 'active'` I think the possible values for care plan activity status is one of those:  `not-started | scheduled | in-progress | on-hold | completed | cancelled | stopped | unknown | entered-in-error` (referencing [here](https://www.hl7.org/fhir/careplan.html))

Reply:
Thanks @achen2401  - fixed now. ... I had been confusing the valueset w/ that for CarePlan.status.